### PR TITLE
Display a default value of an option, if specified

### PR DESCRIPTION
**Pull request by dploeger**
This PR adds the default value of an option to its description, if one is specified.

However, I can not get the test suite to run.

I get  the following errors when running `npm run build`:

```
(...)
$ tslint -c tslint.json -p tsconfig.json --type-check
--type-check is deprecated. You only need --project to enable rule which need type information.
Error at src/core/command/help.ts:188:21: Property 'bold' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:189:42: Property 'dim' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:194:21: Property 'bold' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:204:11: Property 'green' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:223:11: Property 'green' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:224:13: Property 'bold' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:246:19: Property 'bold' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:268:17: Property 'bold' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:286:13: Property 'bold' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:296:11: Property 'green' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:301:11: Property 'green' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:331:19: Property 'bold' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/command/help.ts:337:11: Property 'green' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/core/error.ts:15:27: Property 'dim' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/shim/shim.ts:46:29: Property 'red' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
Error at src/shim/shim.ts:48:29: Property 'red' does not exist on type 'typeof "/Users/dennis.ploeger/Documents/prog/devops/forks/clime/node_modules/chalk/types/index"'.
(...)
```

### CODE REVIEW

**Comment by dploeger** [comment]
Hm. Travis didn't have that issue seemingly...

**Comment by dploeger** [comment]
I'll try again to let it run locally, so I can add a test for the feature.

**Comment by dploeger** [comment]
Okay, done. I basically just accepted the baseman tests, because the default description was already displayed there of course.

File: src/core/command/help.ts [hunk 1]
```diff
@@ -322,6 +323,10 @@ ${buildTableOutput(parameterDescriptionRows, { indent: 4, separators: ' - '})}`)
             triggerStr += ` <${placeholder || key}>`;
           }
 
+          if (defaultValue !== undefined && defaultValue) {
+            description = description ? `description [${defaultValue}]` : `[${defaultValue}]`;
```
Comment (vilic) [inline src/core/command/help.ts]: `defaultValue` could be falsy non-string (e.g. `false`, `0`)

**Comment by vilic** [comment]
Hi, thanks for this PR!

A small problem is that `default` could be falsy non-string value, we may need to update the condition.

And it would be nice if we add that case into tests.

**Comment by dploeger** [comment]
I added some tests and optimized the non-string outputs. I think, it's fine this way. Booleans are reported as "false" or "true" and null is reported as "null".

**Comment by dploeger** [comment]
Hm. Don't know, why travis fails. I don't think, it has to do with my modifications, though, because I didn't get the errors here.

**Comment by vilic** [comment]
@dploeger Hmm, it seems that some baseman reference files newly generated are added to this commit. Did you miss any?

**Comment by dploeger** [comment]
Argh. Damn. I forgot to add the reference files to git. 🤦‍♂️

**Comment by vilic** [comment]
@dploeger Thanks! I may tweak the output format a little bit later, FYI.

**Pull request merged**